app/models/exercise.rb

```diff
@@ -55,6 +55,10 @@ def guide_done_for?(user)
     guide.done_for?(user)
   end
 
+  def choice?
```

Comment:
We were using choices? (checks if choice field is not empty) and I think we should use this one that delegates it to the editor.

Reply:
OK